_schemaV2.graphql

```diff
@@ -13903,9 +13903,6 @@ type Money {
   minor: Long!
 }
 
-# A money object with amount in minor currency and a currency code, or an error object
-union MoneyOrErrorUnion = Error
```

Comment:
Hmm, why the money type removal from gravgraphql, checking

Reply:
`gravity.graphql` isn't important/used by anything, just our dev workflow where MP needs copies of those to generate a `_schema`. The only diff that matters is `_schema`

Reply:
++ true, I was worried about the money related stuff but that seems to be part of the innards of some of this other code!